### PR TITLE
Remove transform for Firefox, Safari

### PR DESCRIPTION
**Pull request by dparikh**

### CODE REVIEW

File: src/ui/ui.css [hunk 1]
```diff
@@ -56,7 +56,6 @@ swg-loading {
   position: absolute !important;
   top: 40% !important;
   left: 45% !important;
-  transform: translate(-50%, -50%) !important;
```
[the remark this answers is not in the document]
Comment (dvoytenko) [reply src/ui/ui.css]: But it'd seem the shift/centering is due to `top:40%; left: 45%` above? That seems weird to center with percent. Does it really work correctly?

File: src/ui/ui.css [hunk 2]
```diff
@@ -56,7 +58,6 @@ swg-loading {
   position: absolute !important;
   top: 40% !important;
```
Comment (dvoytenko) [inline src/ui/ui.css]: Why then keep `top`/`left`? I don't think the content will ever be in the middle this way...

File: src/ui/ui.css [hunk 3]
```diff
@@ -45,18 +47,13 @@ swg-loading-container {
 }
 
 swg-container,
```
Comment (dvoytenko) [inline src/ui/ui.css]: Could you move all these `display:block` to the very top?

File: src/ui/ui.css [hunk 4]
```diff
@@ -20,8 +20,10 @@ body {
 }
 
 swg-loading-container {
-  position: absolute !important;
   width: 100% !important;
+  display: flex !important;
+  align-items: center !important;
+  justify-content: center !important;
   height: 148px !important;
```
Comment (dvoytenko) [inline src/ui/ui.css]: Check if this can be made `100%`, coupled with `min-height`.

Comment (dparikh) [reply src/ui/ui.css]: Done

File: src/ui/ui.css (hunk 3 above)
Comment (dparikh) [reply src/ui/ui.css]: Done. Moved after body {...}

**Pull request merged**